portality/formcontext/forms.py

```diff
@@ -1,17 +1,20 @@
 import re
 from datetime import datetime
 
-from wtforms import Form, validators
+from flask_login import current_user
+from wtforms import Form, validators, StringField, FieldList, FormField, TextAreaField
 from wtforms import StringField, TextAreaField, IntegerField, BooleanField, FormField, FieldList, RadioField
 from wtforms import widgets
 
+from portality import regex
+from portality.core import app
+
 from portality.formcontext.fields import DOAJSelectField, DOAJSelectMultipleField, URLField, TagListField, DisabledTextField, PermissiveSelectField, OptionalRadioField
-from portality.formcontext.validate import URLOptionalScheme, OptionalIf, ExclusiveCheckbox, ExtraFieldRequiredIf, MaxLen, RegexpOnTagList, ReservedUsernames
+from portality.formcontext.validate import URLOptionalScheme, OptionalIf, ExclusiveCheckbox, ExtraFieldRequiredIf, \
+    MaxLen, RegexpOnTagList, ReservedUsernames, ThisOrThat
 
 from portality.formcontext.choices import Choices
 
-from portality.view.forms import ArticleForm, AuthorForm
-
 ISSN_REGEX = re.compile(r'^\d{4}-\d{3}(\d|X|x){1}$')
```

Comment:
Since we're importing `portality.regex` here, we might as well move this over there. I'll do it before the merge.